tests/testthat/test-run-nmtran.R

```diff
@@ -0,0 +1,217 @@
+context("testing NM-TRAN")
+
+# define constants
+MODEL_DIR_BBI <- file.path(dirname(ABS_MODEL_DIR), "test-nmtran-models")
+NONMEM_DIR <- Sys.getenv("BBR_TESTS_NONMEM_DIR", "/opt/NONMEM")
+REQ_NONMEM_VERSIONS <- c("nm73gf", "nm74gf", "nm75", Sys.getenv("BBR_TESTS_NONMEM_VERSION"))
```

Comment:
[ follows [previous thread](https://github.com/metrumresearchgroup/bbr/pull/705#discussion_r1693621070) ]

As with the previous iteration, these tests will fail if `BBR_TESTS_NONMEM_VERSION` or `BBR_TESTS_NONMEM_DIR` is set to something other than the default. The downstream assertions hard code the paths.

My suggestion in the previous thread was to not check these variables at all.

Reply:
They shouldnt _fail_ because of the `skip` call I have after this, but they would be skipped if `BBR_TESTS_NONMEM_VERSION` pointed to a `NONMEM` version we didnt have within `NONMEM_DIR`. Ill remove those environment vars since they dont really help though

Reply:
> They shouldnt _fail_ because [...]

```
$ export BBR_TESTS_NONMEM_VERSION=nm75
$ Rscript -e 'devtools::test(filter = "nmtran")'
ℹ Testing bbr
✔ | F W  S  OK | Context
✖ | 4       28 | testing NM-TRAN [4.3s]                                                                               
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Failure ('test-run-nmtran.R:63:9'): run_nmtran: nmtran_setup: executable and nonmem version
nmtran_specs$nmtran_exe not equal to "/opt/NONMEM/nm74gf/tr/NMTRAN.exe".
1/1 mismatches
x[1]: "/opt/NONMEM/nm75/tr/NMTRAN.exe"
y[1]: "/opt/NONMEM/nm74gf/tr/NMTRAN.exe"

Failure ('test-run-nmtran.R:65:9'): run_nmtran: nmtran_setup: executable and nonmem version
nmtran_specs$nonmem_version not equal to "nm74gf".
1/1 mismatches
x[1]: "nm75"
y[1]: "nm74gf"

Failure ('test-run-nmtran.R:80:9'): run_nmtran: nmtran_setup: executable and nonmem version
nmtran_specs$nmtran_exe not equal to "/opt/NONMEM/nm74gf/tr/NMTRAN.exe".
1/1 mismatches
x[1]: "/opt/NONMEM/nm75/tr/NMTRAN.exe"
y[1]: "/opt/NONMEM/nm74gf/tr/NMTRAN.exe"

Failure ('test-run-nmtran.R:82:9'): run_nmtran: nmtran_setup: executable and nonmem version
nmtran_specs$nonmem_version not equal to "nm74gf".
1/1 mismatches
x[1]: "nm75"
y[1]: "nm74gf"
[...]
[ FAIL 4 | WARN 0 | SKIP 0 | PASS 28 ]

$ git rev-parse HEAD
ebe3f13f41eb1d2ebbabf97473b9ad73b007a700
```

Reply:
Ah I misunderstood by what you meant when you said fail. Yeah I would have had to adjust the expectations in a few spots, but thought you meant they wouldnt be able to hypothetically run because a `NONMEM` installation was missing or something. I deleted that env var though so moot point now